pkg/network/go/bininspect/newproc.go

```diff
@@ -230,9 +231,8 @@ func (i *newProcessBinaryInspector) getGoroutineIDMetadata(abi GoABI) (Goroutine
 			// https://go.googlesource.com/go/+/refs/heads/dev.regabi/src/cmd/compile/internal-abi.md#amd64-architecture
 			runtimeGRegister = 14
 		case GoArchARM64:
-			// https://golang.org/doc/asm#arm64
-			// TODO make sure this is valid
-			runtimeGRegister = 27
+			// https://go.googlesource.com/go/+/refs/heads/master/src/cmd/compile/abi-internal.md#arm64-architecture
+			runtimeGRegister = 28
```

Comment:
I think that has changed over the years
R14 in https://go.googlesource.com/go/+/refs/tags/go1.17/src/cmd/compile/abi-internal.md

Reply:
The page you linked is for amd64. Unfortunately older versions of the ABI documentation doesn't seem to have a arm64, section, but looking at the code you can see that the register with the goroutine pointer has been the same:
https://go.googlesource.com/go/+/refs/tags/go1.14/src/cmd/internal/obj/arm64/a.out.go#241